modules/nested-tabs-html/assets/scss/frontend.scss

```diff
@@ -190,6 +189,15 @@
 				-webkit-text-stroke-color var( --n-tabs-title-transition ),
 				transform var( --n-tabs-title-transition );
 
+			// Remove tap highlight color on iOS: https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-tap-highlight-color.
+			-webkit-tap-highlight-color: transparent;
+			-webkit-touch-callout: none;
+			-webkit-user-select: none;
+			-khtml-user-select: none;
+			-moz-user-select: none;
+			-ms-user-select: none;
+			user-select: none;
```

Comment:
Do we need all the browser prefixes?

I think that the `autoprefixer` script adds them based on our `.browserslistrc` settings.

Reply:
@rami-elementor You are correct. I have removed them.